packages/@n8n/db/tsconfig.json

```diff
@@ -6,7 +6,10 @@
 		"baseUrl": "src",
 		"tsBuildInfoFile": "dist/typecheck.tsbuildinfo",
 		"experimentalDecorators": true,
-		"emitDecoratorMetadata": true
+		"emitDecoratorMetadata": true,
+
+		// remove all options below this line
+		"strictPropertyInitialization": false
 	},
 	"include": ["src/**/*.ts"]
 }
```

Comment:
we could add project references here to improve the development experience.

Reply:
Agree, let me do this in the next one where we'll be populating this package properly 👍🏻